features/admin.branding.v1/components/branding-core.tsx

```diff
@@ -248,7 +248,8 @@ const BrandingCore: FunctionComponent<BrandingCoreInterface> = (
                     layout: predefinedLayouts
                 }
             ));
-            setSelectedLayout(originalBrandingPreference.preference.layout.activeLayout);
+
+            setSelectedLayout(brandingPreference?.layout?.activeLayout);
```

Comment:
Lets assign `BrandingPreferenceUtils.migrateLayoutPreference(...` to a var and use in the `setSelectedLayout`.

Since `setBrandingPreference` is async, accessing the just set state in the next line will be unpredictable. And might cause regressions if the Default value set in the useState changes.

Reply:
Fixed with commit https://github.com/wso2/identity-apps/pull/6364/commits/7f5533f8b1b5657863da21d51a087d40f79c42a3